src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.Runtime.Augments;
+using System;
+using System.Reflection;
+
+namespace System
+{
+    public partial class AppDomain
+    {
+        private static AppDomain domain = new AppDomain();
+        private AppDomain() {}
+        public static AppDomain CurrentDomain => domain;
+        public Assembly[] GetAssemblies() => AppDomainAugments.GetAssemblies();
+        public string BaseDirectory => "";
+        public string RelativeSearchPath => "";
```

Comment:
These actually do not make sense for coreclr. They were present in AppDomain for desktop wherein you have AppDomainSetup & AppDomainManager to set these values. So in the absense of AppDomainSetup & AppDomainManager for coreclr you cannot actually change values for BaseDirectory & RelativeSearchPath and hence it will always return empty string. We are exposing these properties (even though it returns epmpty string) for binary compat. But I also see a similar property in AppContext https://github.com/dotnet/corefx/blob/f5cbdc34f4475ddb62e327227d2841048a1770c2/src/System.AppContext/ref/System.AppContext.cs#L13. So now it does not make sense to add this here as well. @danmosemsft what do you think?